src/Pages/Signup/Signup.js

```diff
@@ -0,0 +1,347 @@
+        }
+      });
+  };
+
+  //pw 유효성검사 (알림창띄우기)
+  pwCheck = e => {
+    const pwreg =
+      /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[$@$!%*?&])[A-Za-z\d$@$!%*?&]{8,18}/g;
+    let userPw = e.target.value;
+    this.setState({ password: e.target.value });
+
+    if (false === pwreg.test(userPw)) {
+      alert('다시 입력해주세요. (대소문자,숫자,특수문자 모두 포함 8~18자)');
+    }
+  };
+  //pw 중복체크
+  handleConfirmPassword = e => {
+    this.setState(
+      {
+        [e.target.name]: e.target.value,
+      },
+      () => {
+        const { password, rePassword } = this.state;
+
+        let isAvailedPassword = '';
+
+        if (e.target.value) {
+          isAvailedPassword =
+            password === rePassword ? '' : '비밀번호가 일치하지 않습니다.';
+        }
+
+        this.setState({ isAvailedPassword });
+      }
+    );
+  };
+
+  //id , name , phone , email, address value값 지정 / 주소 value 합치기
+  handleInputChange = e => {
+    this.setState(
+      {
+        [e.target.name]: e.target.value,
+      },
+      () => {
+        if (e.target.name === 'secondAddress') {
+          this.setState({
+            address: this.state.firstAddress + this.state.secondAddress,
+          });
+        }
+      }
+    );
+  };
+
+  //주소
+  handleOpenPost = () => {
+    this.setState({
+      isDaumPost: true,
+    });
+  };
+  // daum api 불러오기
+  handleAddress = data => {
+    let allAddress = data.address;
+    let extraAddress = '';
+    // let zoneCodes = data.zonecode;
+    if (data.addressType === 'R') {
+      if (data.bname !== '' && /[동|로|가]$/g.test(data.bname)) {
+        extraAddress += data.bname;
+      }
+      if (data.buildingName !== '') {
+        //공동주택제한일 경우 if안에  && data.apartment === 'Y' 추가
+        extraAddress +=
+          extraAddress !== '' ? ', ' + data.buildingName : data.buildingName;
+        // extraAddress !== '' ? `, ${data.buildingName}` : data.buildingName;
+      }
+      allAddress += extraAddress !== '' ? `(${extraAddress})` : ''; //템플릿 리터럴 표현
+    }
+    this.setState({
+      firstAddress: allAddress,
+    });
+  };
+
+  //회원가입버튼
+  signUpSummit = e => {
+    e.preventDefault();
+    if (this.state.identity === '') {
+      alert('아이디를 입력해주세요');
+    } else if (this.state.password === '') {
+      alert('비밀번호를 입력해주세요');
+    } else if (this.state.name === '') {
+      alert('이름을 입력해주세요');
+    } else if (this.state.phone === '') {
+      alert('휴대폰 번호를 입력해주세요');
+    } else if (this.state.email === '') {
+      alert('이메일을 입력해주세요');
+    } else if (this.state.address === '') {
+      alert('주소를 입력해주세요');
+    } else {
+      this.clickSignup();
+    }
+  };
+
+  //백엔드 통신
+  clickSignup = e => {
+    e.preventDefault();
+    fetch('http://10.58.2.234:8000/user/sign-up', {
+      method: 'POST',
+      body: JSON.stringify({
+        identity: this.state.identity,
+        password: this.state.password,
+        name: this.state.name,
+        phone: this.state.phone,
+        email: this.state.email,
+        address: this.state.address,
```

Comment:
- address 라는 스테이트는 삭제해 주시고,
```suggestion
        address: this.state.firstAddress + this.state.secondAddress,
```